rita_common/src/payment_controller/mod.rs

```diff
@@ -252,32 +271,49 @@ async fn make_payment(mut pmt: PaymentTx) -> Result<(), PaymentControllerError>
         attempt: 0u8,
     };
 
-    match neigh_ack {
-        Ok(mut val) => {
-            if val.status().is_success() {
-                info!(
-                    "Payment pmt with txid: {:#066x} is sent to our neighbor with status {:?} and body {:?} via url {}, using full node {} and amount {:?}",
-                    tx_id,
-                    val.status(),
-                    val.body().await,
-                    neighbor_url,
-                    full_node,
-                    pmt.amount
-                );
-            } else {
-                error!(
-            "We published txid: {:#066x} but our neighbor responded with status {:?} and body {:?}, will retry",
-            tx_id, val.status(), val.body().await);
-                queue_resend(resend_info);
-            }
+    // Hit both endpoints, in the case of a node having both endpoints, validator will simply get a duplicate transaction txid and discard it.
+    let (neigh_ack_v2, neigh_ack) = join(neigh_ack_v2, neigh_ack).await;
+
+    if let Ok(mut val) = neigh_ack_v2 {
+        if val.status().is_success() {
+            info!(
+                "Payment pmt with txid: {:#066x} is sent to our neighbor with status {:?} and body {:?} via url {}, using full node {} and amount {:?}",
+                tx_id,
+                val.status(),
+                val.body().await,
+                neighbor_url_v2,
+                full_node,
+                pmt.amount
+            );
+        } else {
+            error!(
+        "We published txid: {:#066x} via url {} but our neighbor responded with status {:?} and body {:?}",
```

Comment:
would you do a match on the tuple with both results and print only one message even if both succeed? We don't want to print extra log messages if possible becuase we parse these later on in graylog

Reply:
Fixed